tests/template_tests/test_origin.py

```diff
@@ -24,3 +24,7 @@ def test_origin_compares_not_equal(self):
         # Use assertIs() to test __eq__/__ne__.
         self.assertIs(a.origin == b.origin, False)
         self.assertIs(a.origin != b.origin, True)
+
+    def test_origin_repr(self):
+        _, origin = self.engine.find_template('index.html')
+        self.assertEqual(repr(origin), f'Origin(name={TEMPLATE_DIR}/index.html)')
```

Comment:
```suggestion
        self.assertEqual(repr(origin), f'Origin(name=“{TEMPLATE_DIR}/index.html”)')
```

Reply:
Also here,``f'Origin(name=“{TEMPLATE_DIR}/index.html”)'`` needs to be cross-platform. I think ``os.path.join`` will do the trick!